examples/region_of_interest/ExtractLineProfile.py

```diff
@@ -0,0 +1,57 @@
+"""
+Extract line profile from image
+===============================
+
+Use a ``Line2DROI`` to interactively extract a line profile with a certain width
+from an image.
+
+"""
+
+#%%
+# Initialize image data as HyperSpy signal:
+import hyperspy.api as hs
+import scipy
+img = hs.signals.Signal2D(scipy.datasets.ascent())
```

Comment:
Not sure if this is necessary but to avoid ambiguity between values/indices, it can be useful to set a scale to the signal axes:

```suggestion
import scipy
img = hs.signals.Signal2D(scipy.datasets.ascent())

for axis in img.axes_manager.signal_axes:
    axis.scale = 0.1

```